DTS-STN/DECD-Design-System
# Tailwind3

Updating tailwind to version 3

**Reminder: Add all changes for this PR to the CHANGELOG.md**

Main Reviewer: @jvaddinen 

#### Description & Background Context: 
Updated tailwind version and other dependencies to be up to date.
Made minor changes to repo and cleaned up code

#### Where should the reviewer start?

File name: tailwind.config.js

#### How should this be tested?
Test components on storybook branch and in another project.
Confirm that the version update didn't break any of our code

#### Are there any breaking changes?
No
